packages/dx-react-grid/docs/reference/grid.md

```diff
@@ -34,6 +35,8 @@ A value with the following shape:
 Field | Type | Description
 ------|------|------------
 name | string | Specifies the field name in the data row to obtain a column value. A unique key can also be used to identify a column
+getCellData | (row: [Row](#row), columnName: string) => any | Specifies the function used to get a cell data
+createRowChange | (row: [Row](#row), columnName: string, value: string &#124; number) => object | Specifies the function used to create a row change
```

Comment:
Also seems that the `columnName` argument should be the last because it will not be frequently used